app/src/main/java/org/fossasia/openevent/general/ticket/TicketsViewModel.kt

```diff
@@ -43,6 +43,7 @@ class TicketsViewModel(
     private val mutableTicketTableVisibility = MutableLiveData<Boolean>()
     val ticketTableVisibility: LiveData<Boolean> = mutableTicketTableVisibility
     val ticketIdAndQty = MutableLiveData<List<Triple<Int, Int, Float>>>()
+    var discountCodeCurrentLayout = APPLY_DISCOUNT_CODE
```

Comment:
Why is this var?

Reply:
to retain current discount code layout on screen rotation or coming back from AttendeeFragment